documentation/how-to/confirmation.md

```diff
@@ -0,0 +1,173 @@
+# Confirmation How-To
+
+This is a quick tutorial on how to configure your application to enable confirmation.
```

Comment:
An important bit of the docs refactor is making sure that each thing properly adheres to its "type". This does in fact appear to be a tutorial to me, and so this should probably go in `/tutorials`. 

I've found that there is actually almost zero "how-to" documentation according to diataxis in Ash. There is explanation, and tutorials, and reference, but not so much "how-to". (just a meta comment).